packages/twenty-server/src/engine/metadata-modules/field-metadata/validators/is-field-metadata-default-value.validator.ts

```diff
@@ -50,6 +50,12 @@ export class IsFieldMetadataDefaultValue
           where: {
             id,
           },
+          relations: [
+            'fromRelationMetadata',
```

Comment:
Those relations are not used in this file I believe. Let's not query relations if they are not used